libvirt/httpd/httpd.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+TARGET_PORT=8080
+
+dnf install httpd
+sed -i "s/Listen 80$/Listen ${TARGET_PORT}/" /etc/httpd/conf/httpd.conf 
+service httpd start
+virsh -c qemu+tcp:///system pool-define-as --name httpd --type dir --target /var/www/html
+virsh -c qemu+tcp:///system pool-autostart httpd
+virsh -c qemu+tcp:///system pool-start httpd
+
+firewall-cmd --zone libvirt --add-port ${TARGET_PORT}/tcp --permanent
```

Comment:
This keeps it open only for the libvirt zone? if so, great. Can you access externally? from another host? if so, I'd recommend you restrict the httpd port further.

Reply:
It opens it for ports within the libvirt zone. You can access it from any bridge connected to that zone. In this case, any bridge create by libvirtd.

External access *should* be across the public network link, so that traffic should be disallowed. Other hosts would only be able to hit that endpoint if the are accessing over a link created by libvirt.